### ISSUE TITLE
vagrant up throws error

### ISSUE THREAD

**Issue by mahsohn** [pelias/vagrant#42]
I followed the article for setting up pelias, I installed virtualbox and vagrant via apt-get and chef-dk from the website. when I run vagrant up I get the following error:

Failed to load the "vagrant-berkshelf" plugin. View logs for more details.

Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

Vagrant:
* Unknown configuration section 'berkshelf'.

here is some version info if it helps

lsb_release
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.3 LTS
Release:        14.04
Codename:       trusty

vagrant --version
Vagrant 1.4.3

vagrant plugin list
vagrant-berkshelf (4.1.0)
vagrant-omnibus (1.4.1)

berks --version
4.3.2

do I need to use a particular vagrant version?
I've read online that berkshelf is no longer a plugin, any help would be appreciated

**Comment by orangejulius** [comment]
Hey @mahsohn,
I've always found chef Vagrant images a bit difficult to set up, so I'm sorry you're seeing issues too.

On my machine which does currently work, I'm running Vagrant 1.7.4 and berks 3.3.0, but more or less the same versions of the plugins.

Can you try with Vagrant 1.7? I have a feeling that might help.

**Comment by mahsohn** [comment]
Thanks, installing the latest vagrant from their site seemed to work... however I'm getting another error.

The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'poweroff' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.



I'm trying to test this out on an EC2 instance but I am assuming that is the problem.
Can pelias be run on an amazon instance or does it need a dedicated server?

**Comment by orangejulius** [comment]
Yeah, that looks like a Virtualbox configuration error. My suggestion for debugging those is to actually open up the VirtaulBox GUI (like the message says), and test with a simpler Vagrant image. The Windows images from http://modern.ie work great for that.
